rust/gui-client/src-tauri/src/client/gui/os_linux.rs

```diff
@@ -35,17 +35,20 @@ pub(crate) async fn set_autostart(enabled: bool) -> Result<()> {
 /// Since clickable notifications don't work on Linux yet, the update text
 /// must be different on different platforms
 pub(crate) fn show_update_notification(
+    app: &AppHandle,
     _ctlr_tx: super::CtlrTx,
     title: &str,
     download_url: url::Url,
 ) -> Result<()> {
-    show_notification(title, download_url.to_string().as_ref())?;
+    show_notification(app, title, download_url.to_string().as_ref())?;
     Ok(())
 }
 
 /// Show a notification in the bottom right of the screen
-pub(crate) fn show_notification(title: &str, body: &str) -> Result<()> {
-    Notification::new(BUNDLE_ID)
+#[allow(clippy::unnecessary_wraps)]
```

Comment:
```suggestion
#[expect(clippy::unnecessary_wraps)]
```

I'd suggest we prefer `expect` over `allow` everywhere so we get a warning when it is no longer necessary.